src/main/java/io/leonis/subra/game/data/Player.java

```diff
@@ -109,8 +127,11 @@ public State(
     }
   }
 
+  /**
+   * The {@link io.leonis.zosma.game.Identity identity} of the robot.
+   */
   @Value
```

Comment:
Unrelated to this PR; but do we actually want accessors for the data in identities ?

Reply:
it's not required since the interface should cover all necessary accessors. i don't care either way

Reply:
I think we should change all Identity-classes to just have an `@EqualsAndHashcode` with a finialized class. Will make an PR as soon as this is merged.